nowcast/delphi_nowcast/nowcast_fusion/covariance.py

```diff
@@ -0,0 +1,213 @@
+"""
+========================================================================
+THIS CODE IS COPIED FROM
+https://github.com/cmu-delphi/nowcast/blob/main/src/fusion/covariance.py
+========================================================================
+
+Maximum likelihood covariance estimation that is robust to insufficient and
+missing values.
+"""
+
+# standard library
+import abc
+
+# third party
+import numpy as np
+import scipy.linalg
+import scipy.stats
+
+# first party
+from .opt_1d import maximize
+
+
+def nancov(X):
+  """
```

Comment:
This file appears to use 2-space indents -- I'm surprised to find that lints?

Reply:
i ignored them all of davids files since so I wouldn't have to modify them much. I can unignore + fix linting if you think thats better

Reply:
nah, ignore is fine, that'll make it easier to distribute patches across all usages if we ever have to